### PR TITLE
Add accounts templates for fr_BE

### PR DESCRIPTION
**Pull request by bigon**
Hello,

This adds the accounts templates for fr_BE (French Belgium)

It's almost a copy of the fr_FR templates. There are still some localization to do (ie the name of some organizations or taxes, like URSSAF) but it should be a good base already

### CODE REVIEW

**Comment by gjanssens** [comment]
Hi, thanks for your work. As I'm from Belgium as well (though from the Dutch part) I'm obviously quite pleased to see the Belgian localization get more attention :)

Your changes are technically correct and the PR is clear and well-formed. Do you intend to add more changes to this PR (as you hint there are still some localization issues) or is it meant to be merged as is ?

File: data/accounts/fr_BE/acctchrt_common.gnucash-xea [hunk 1]
```diff
@@ -0,0 +1,754 @@
...
+  xmlns:tte="http://www.gnucash.org/XML/tte"
+  xmlns:vendor="http://www.gnucash.org/XML/vendor">
+    <gnc-act:title>
+      Comptes communs
+    </gnc-act:title>
+    <gnc-act:short-description>
+      Un ensemble de base de comptes les plus habituellement utilisés
+    </gnc-act:short-description>
+    <gnc-act:long-description>
+    La plupart des utilisateurs devront sélectionner cet ensemble de comptes.  Il comprend la plupart des comptes communément utilisés (courant, épargne, liquidités, carte de crédit, revenu, dépenses habituelles).
```
Comment (fellen) [inline data/accounts/fr_BE/acctchrt_common.gnucash-xea]: This line was updated in the english version:
https://github.com/Gnucash/gnucash/commit/dddc278851eb1bff3ec5c35cea282a69a3e4d05c

Can you apply the change to all french common files?

**Comment by fellen** [comment]
Thanks for the promt fix!
So only Geerts question remains:
> Do you intend to add more changes to this PR (as you hint there are still some localization issues) or is it meant to be merged as is ?

**Comment by bigon** [comment]
Hello,

I think that can be merged for now, the rest can be done later I guess

**Comment by fellen** [comment]
Sure, done.

**Pull request merged**